qml/StyledToolBar.qml

```diff
@@ -147,7 +147,9 @@ ToolBar {
                 selectedTextColor: defaultBackground
                 Layout.fillWidth: true
                 Layout.fillHeight: true
-                placeholderText: qsTr("Quick find")
+                placeholderText: !!(navigator.currentItem && navigator.currentItem.objectName !== 'settingsView')
+                                 ? qsTr("Quick find")
+                                 : qsTr("Quick find in Settings")
```

Comment:
Sentence case? Maybe just "Search accounts" and "Search settings"?

Reply:
agreed, fixed!